{{ cookiecutter.package_name|slugify }}/pyproject.toml

```diff
@@ -158,7 +159,7 @@ convention = "{{ cookiecutter.docstring_style|lower }}"
 
 # https://docs.pytest.org/en/latest/reference/reference.html#ini-options-ref
 [tool.pytest.ini_options]
-addopts = "--color=yes --doctest-modules --exitfirst --failed-first --strict-config --strict-markers --verbosity=2 --junitxml=reports/pytest.xml"
+addopts = "--color=yes --doctest-modules --exitfirst --failed-first --strict-config --strict-markers --typeguard-packages={{ cookiecutter.package_name|slugify(separator='_') }} --verbosity=2 --junitxml=reports/pytest.xml"
```

Comment:
We'll probably have to change this line once #79 lands. But fine for now 😃

Reply:
Updated!